c-svc23Vectors/math.c

```diff
@@ -34,3 +36,94 @@ int dotProductVectors(const int *input1, const int *input2, int length) {
 int dotProduct3dVectors(const int input1[3], const int input2[3]) {
     return dotProductVectors((const int*)input1, (const int*)input2, 3);
 }
+
+double getBiggestNumber(double axisArray[], int length){
+    double highestNumber = 0;
+    int negativeOrPositive = 1;
+    int i;
+    for (i = 0; i < TIME_SERIES_VECTORS_AMOUNT; i++)
+    {
+        double currentNumber = axisArray[i];
+        if(currentNumber > 0)
+        {
+            if (currentNumber > highestNumber)
+            {
+                highestNumber = currentNumber;
+                negativeOrPositive = 1;
+            }
+            
+        } else
+        {
+            if (-1 * currentNumber > highestNumber)
+            {
+                highestNumber = -1 * currentNumber;
+                negativeOrPositive = -1;
+            }
+            
+        }
+    }
+    return negativeOrPositive * highestNumber;
+}
+
+double getSmallestNumber(double axisArray[], int length){
```

Comment:
Abstract over biggest- and smallestNumber, as they are almost the same implementation. Maybe you can make an auxiliary function?